tests/distributions/test_distributions.py

```diff
@@ -108,7 +108,10 @@ def test_score_errors_non_broadcastable_data_shape(dist):
         if dist.get_test_distribution_name() == 'LKJCholesky':
             pytest.skip('https://github.com/pytorch/pytorch/issues/52724')
         shape = d.shape()
-        non_broadcastable_shape = (shape[0] + 1,) + shape[1:]
+        if dist.get_test_distribution_name() == 'SineBivariateVonMises':
+            non_broadcastable_shape = (shape[0] + 1,) + (shape[1] + 1,)
+        else:
+            non_broadcastable_shape = (shape[0] + 1,) + shape[1:]
```

Comment:
You shouldn't need to special case here. I believe this test is intended to find broadcasting errors among batch shapes, whereas the `(shape[1] + 1,)` is a more fundamental error in event shape.  If you'd like to test event shape errors, you could add a bespoke test.

Reply:
Thanks for clarifying; I'll move the test to `test_sine_bivariate_von_mises`.